OpenSim/Simulation/Model/Model.cpp

```diff
@@ -105,10 +106,39 @@ Model::Model(const string &aFileName) :
 
     _fileName = aFileName;
     cout << "Loaded model " << getName() << " from file " << getInputFileName() << endl;
+
+    try {
+        finalizeFromProperties();
+    }
+    catch(const InvalidPropertyValue& err) {
+        std::cout << "WARNING: Model was unable to finalizeFromProperties "
+            "due to: " << err.what() <<
+            "\nUpdate the model file and reload OR update the property and call "
+            "finalizeFromProperties() on the model." << endl;
```

Comment:
Minor hygienic issue: inconsistent use of `std::cout` on one side and simply `endl` on the other.